app/views/how-tos/filters.html

```diff
@@ -0,0 +1,108 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+Filters - NHS prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-three-quarters">
+    <h1>Filters</h1>
+    <p class="nhsuk-lede-text">
+      Change how answers are shown.
```

Comment:
This feels a bit overly-specific? Whilst that’s a good use case for filters, it’s not the only use case?

Reply:
Changed to 'change how text is displayed'

Reply:
It's still quite specific - there's lots of uses beyond changing how text is displayed.